data-prepper-plugins/flatten-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/flatten/FlattenProcessorConfig.java

```diff
@@ -28,12 +30,18 @@ public class FlattenProcessorConfig {
     @JsonProperty("source")
     @JsonPropertyDescription("The source key on which to perform the operation. If set to an empty string (<code>\"\"</code>), " +
             "then the processor uses the root of the event as the source.")
+    @ExampleValues({
+        @Example(value = "key2", description = "The flatten operation is performed on the source key equivalent to 'key2'")
+    })
     private String source;
 
     @NotNull
     @JsonProperty("target")
     @JsonPropertyDescription("The target key to put into the flattened fields. If set to an empty string (<code>\"\"</code>) " +
             "then the processor uses the root of the event as the target.")
+    @ExampleValues({
+        @Example(value = "flattened-key2", description = "The flattened fields from the source key specified are put into the specified target key.")
```

Comment:
Maybe mention the example key you give, e.g.:
```suggestion
        @Example(value = "flattened-key2", description = "The flattened fields from the source key specified are put into a field named 'flattened-key2'.")
```